libcodechecker/server/api/report_server.py

```diff
@@ -76,7 +76,7 @@ def wrapper(*args, **kwargs):
             LOG.warning(rf.message)
             raise
         except Exception as ex:
-            LOG.warning(ex.message)
+            LOG.warning(str(ex))
             raise shared.ttypes.RequestFailed(shared.ttypes.ErrorCode.GENERAL,
                                               ex.message)
```

Comment:
Shouldn't we return the whole exception in the `RequestFailed` exception below? In case there is no message attribute raising the other type of exception will fail too.